Porting/manicheck

```diff
@@ -4,28 +4,57 @@
 #  a) files listed in MANIFEST which don't exist
 #  b) files which exist but which aren't in MANIFEST
 
-use strict;
+use v5.14;
 use warnings;
 use File::Find;
+use Getopt::Long;
+use constant SKIP => 125;
+
+my $exitstatus;
+GetOptions('exitstatus!', \$exitstatus)
+    or die "$0 [--exitstatus]";
+
+my %files;
+my $missing = 0;
+my $bonus = 0;
 
 open my $fh, '<', 'MANIFEST' or die "Can't read MANIFEST: $!\n";
-my @files = map { (split)[0] } <$fh>;
-close $fh;
-for (@files) {
-    print "$_ from MANIFEST doesn't exist\n" if ! -f;
+for my $line (<$fh>) {
+    my ($file) = $line =~ /^(\S+)/;
+    ++$files{$file};
+    next
+        if -f $file;
```

Comment:
Why is the `next` statement broken into two lines?

Reply:
> Why is the `next` statement broken into two lines?

Because I was writing it in the style i prefer. Which is inconsistent with the rest of the file.

Oops, well spotted. Thanks for catching this.

Fixed in the rebase.